src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/condition-recommendations.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Get condition recommendations
+metaDescription: For APM entities, you can see and create recommended alert conditions.
+watermark: Limited release
+---
+
+Use the New Relic Navigator's mini-overview to get alert condition recommendations for your monitored entities. With these recommendations, you can create alerts conditions to be notified when your entities aren't performing as they should.
```

Comment:
@paperclypse, I'm not sure what the convention is in alerts, but I'm seeing in this sentence both "alert conditions" and "alerts conditions."

Also, in the second sentence, it almost seemed like it was the alert that were to be notified. How about a more direct construction: "...you can create alerts conditions that notify you when your entities...."?

Reply:
It should be "alerts conditions" or just "conditions". Just a typo on my part.